homeassistant/components/philips_js/media_player.py

```diff
@@ -192,45 +176,50 @@ def is_volume_muted(self):
 
     def turn_on(self):
         """Turn on the device."""
-        if self._on_script:
-            self._on_script.run(context=self._context)
-            self._update_soon(DELAY_ACTION_ON)
+        _LOGGER.debug("Firing event to turn on device")
```

Comment:
Suspect you don't actually need/want this log statement.

Reply:
I don't need it. But it's to help users to figure out why the turn_on feature isn't doing anything.